### PR TITLE
Add new k8s versions

### PR DESCRIPTION
**Pull request by ryansann**

### CODE REVIEW

File: rke/k8s_rke_system_images.go [hunk 1]
```diff
@@ -2438,6 +2438,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band after v2.4.3
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.16.11-rancher1-1": {
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2443]: Update this to `v1.16.10-rancher2-2`until we get the new hyperkube image.

File: rke/k8s_rke_system_images.go [hunk 2]
```diff
@@ -2438,6 +2438,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
...
+			Etcd:                      m("quay.io/coreos/etcd:v3.3.15-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.16.11-rancher1"),
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2445]: This image doesn't exist yet, right? Can we make this `v1.16.10-rancher2`?

File: rke/k8s_rke_system_images.go [hunk 3]
```diff
@@ -2737,6 +2772,40 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.17.7-rancher1-1": {
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2776]: Update this to `v1.17.6-rancher2-2` until we get the new hyperkube image.

File: rke/k8s_rke_system_images.go [hunk 4]
```diff
@@ -2737,6 +2772,40 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
...
+			Etcd:                      m("quay.io/coreos/etcd:v3.4.3-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.17.7-rancher1"),
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2778]: Update this to v1.17.6-rancehr2 until we can get a new hyperkube image

File: rke/k8s_rke_system_images.go [hunk 5]
```diff
@@ -2438,6 +2438,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled out of band after v2.4.3
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2441]: This should be updated as `//Enabled with v2.4.5`

File: rke/k8s_rke_system_images.go [hunk 6]
```diff
@@ -2737,6 +2772,40 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2775]: There should be a comment with `//Enabled with v2.4.5`

File: rke/k8s_rke_system_images.go [hunk 7]
```diff
@@ -2772,6 +2841,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.3"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.7"),
 		},
+		// Enabled in Rancher v2.4.4
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2844]: Change this to `//Enabled in Rancher v2.4.5`

File: rke/k8s_rke_system_images.go [hunk 8]
```diff
@@ -2772,6 +2841,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
...
+		// Reminder: This template contains Nodelocal image which isn't in templates before v2.3.6
+		"v1.18.4-rancher1-1": {
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2846]: Update this to `v1.18.3-rancher2-2`

File: rke/k8s_rke_system_images.go [hunk 9]
```diff
@@ -2772,6 +2841,41 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
...
+			Etcd:                      m("quay.io/coreos/etcd:v3.4.3-rancher1"),
+			Kubernetes:                m("rancher/hyperkube:v1.18.4-rancher1"),
```
Comment (deniseschannon) [inline rke/k8s_rke_system_images.go:2848]: Update this to `v1.18.3-rancher2`

File: rke/k8s_version_info.go [hunk 10]
```diff
@@ -120,6 +120,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 			MinRKEVersion:     "1.1.2-rc0",
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.16.11-rancher1-1": {
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:124]: This will need to be updated to the new template versions

File: rke/k8s_version_info.go [hunk 11]
```diff
@@ -120,6 +120,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
...
+			MinRancherVersion: "2.4.4-rc0",
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:125]: Update this to 2.4.5-rc0 as we are going to introduce @luthermonson's windows 1909 build support so these templates will only be applicable to 2.4.5+

File: rke/k8s_version_info.go [hunk 12]
```diff
@@ -120,6 +120,12 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
...
+			MinRKEVersion:     "1.1.2-rc0",
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:126]: Update this to 1.1.3-rc0 to align with 2.4.5-rc0

File: rke/k8s_version_info.go [hunk 13]
```diff
@@ -144,7 +150,19 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
 		},
 		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
 		// This version includes nodelocal dns only available in RKE v1.0.7 and up
-		"v1.18.3-rancher2-1": {
+		"v1.17.7-rancher1-1": {
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:153]: This will need to be updated to the new template versions

File: rke/k8s_version_info.go [hunk 14]
```diff
@@ -144,7 +150,19 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
...
+			MinRancherVersion: "2.4.4-rc0",
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:154]: Update this to 2.4.5-rc0 as we are going to introduce @luthermonson's windows 1909 build support so these templates will only be applicable to 2.4.5+

File: rke/k8s_version_info.go [hunk 15]
```diff
@@ -144,7 +150,19 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
...
+			MinRKEVersion:     "1.1.2-rc0",
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:155]: Update this to 1.1.3-rc0 to align with 2.4.5-rc0

File: rke/k8s_version_info.go [hunk 16]
```diff
@@ -144,7 +150,19 @@ func loadK8sVersionInfo() map[string]v3.K8sVersionInfo {
...
+		},
+		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.18.3-rancher1-1": {
+			MinRancherVersion: "2.4.5-rc0",
+			MinRKEVersion:     "1.1.3-rc0",
+		},
+		// The Calico/Canal template in this version use functions that are only available in RKE v1.0.0 and up
+		// This version includes nodelocal dns only available in RKE v1.0.7 and up
+		"v1.18.4-rancher2-1": {
```
Comment (deniseschannon) [inline rke/k8s_version_info.go:165]: This will need to be updated to the new template versions

File: rke/k8s_rke_system_images.go (hunk 2 above)
Comment (ryansann) [reply rke/k8s_rke_system_images.go:2445]: Yea, I think I misunderstood this process. I thought we weren't merging until the cve is released, so yea those images don't exist, they were just placeholders for what will be the new hyperkube image. I'll change it

**Pull request merged**